dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit unpacker cmake-multilib
+
+MY_P="deviceatlas-enterprise-c-${PV}"
+
+DESCRIPTION="API to detect devices based on the User-Agent HTTP header"
+HOMEPAGE="https://deviceatlas.com"
+SRC_URI="https://deviceatlas.com/download/api/enterprise/c -> ${MY_P}.zip"
+
+LICENSE="public-domain"
```

Comment:
`<hat type="Licenses">`
I can't find any actual license text on their website either

 all-rights-reserved is available in tree:
`LICENSE=all-rights-reserved`.
You also must add  `RESTRICT='fetch mirror bindist'`
`</hat>`